kuronometer-core/src/main/scala/com/github/pedrovgs/kuronometer/free/interpreter/api/KuronometerApiClientConfig.scala

```diff
@@ -0,0 +1,10 @@
+package com.github.pedrovgs.kuronometer.free.interpreter.api
+
+case class KuronometerApiClientConfig(
```

Comment:
Something like this would help to not hardcode those default config values here. https://github.com/47deg/case-classy